### PR TITLE
add meta tags

### PR DESCRIPTION
**Pull request by 7ORP3DO**
Add basic meta tags.

linked Issue: #219

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,12 +1,22 @@
...
     <meta name="theme-color" content="#e30062" />
 
     <title>Welcome to Real Dev Squad</title>
 
+    <meta name="title" content="Real Dev Squad" />
+    <meta
+      name="description"
+      content="Real Dev Squad is a community of developers, where we come together to learn, use that in building projects, prepare for interviews, help each-other, put those projects online and try to help the set of new developers or freshers, helping them get better jobs and higher salaries with quality work."
+    />
+    <meta name="keywords" content="RDS, developers,Real Dev Squad" />
```
Comment (Rucha1499) [inline index.html:15]: I came across some blogs/ docs which state that the meta tag `keywords` is not currently used as it is not considered for page ranking and has no SEO value as such. For reference -> https://developers.google.com/search/blog/2009/09/google-does-not-use-keywords-meta-tag

should will still consider adding it? what are your thoughts on it?

**Review by Rucha1499 — COMMENTED** [review]
Thank you @7ORP3DO for the PR! Good job here. 🔥 

Adding some comments, please have a look at them

**Review by ankushdharkar — CHANGES_REQUESTED** [review]
@7ORP3DO Could you address the comments and resolve them please

**Comment by Rucha1499** [comment]
Closing this PR due to inactivity.

**Pull request closed without merging**